extension/src/GodotT5Glasses.cpp

```diff
@@ -21,6 +23,67 @@ GodotT5Glasses::GodotT5Glasses(std::string_view id)
     set_swap_chain_size(g_swap_chain_length);
 }
 
+
+Transform3D GodotT5Glasses::get_head_transform() {
+	Quaternion orientation;
+	Vector3 position;
+	get_glasses_orientation(orientation.x, orientation.y, orientation.z, orientation.w);
+	get_glasses_position(position.x, position.y, position.z);
+
+	Transform3D headPose;
+    headPose.set_origin(position);
+	headPose.set_basis(orientation.inverse());
+    headPose.rotate(Vector3(1,0,0), -Math_PI / 2.0f);
```

Comment:
Not sure if this will work, it'll change left to right. The image in OpenGL is flipped upside down compared to Vulkan (it's actually OpenGL that is "wrong"). Rotating does too much and changes the orientation of things, flipping during rendering has all sorts of nasty results.

This might be something to take up with Kasper because other implementations using Vulkan will run into this as well.

Reply:
That code is changing the head pose from T5's coordinate system to Godot's.

Reply:
ah ok, I was drawing conclusions based on our discussion of yesterday, gotcha